src/components/grid-view/GridView.scss

```diff
@@ -33,7 +33,7 @@ $columns5prct: calc(20% - 8px); /* 100%/5 - 40px/5 */
         margin: 0;
     }
 
-    .table-row {
+    .bdl-GridView-tableRow {
```

Comment:
We should also take the opportunity to get rid of the global style variables at the top of this file